### PR TITLE
`Communication`: Don't show preview images if there are multiple links in a message

### PR DESCRIPTION
**Pull request by pellumbbaboci**
#### General

- [ ] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [ ] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/development-process/#naming-conventions-for-github-pull-requests).

#### Client
- [x] **Important**: I implemented the changes with a very good performance, prevented too many (unnecessary) REST calls and made sure the UI is responsive, even with large data.
- [x] I followed the [coding and design guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/client/).
- [x] Following the [theming guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/client-design/), I specified colors only in the theming variable files and checked that the changes look consistent in both the light and the dark theme.
- [x] I added multiple integration tests (Jest) related to the features (with a high test coverage), while following the [test guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/client-tests/).
- [x] I documented the TypeScript code using JSDoc style.
- [x] I added multiple screenshots/screencasts of my UI changes.

### Motivation and Context

In PR #6616 link preview was introduced and the preview includes an image. But when multiple links are sent in a message the previews can get quite big for a single message. An suggestion from @bassner  was to remove the images and only have the Title and Description when there are multiple links in a message.
### Description

In new indicator is introduced to differentiate if a message includes multiple or single url. And if there are multiple the preview image tag is not shown.

### Steps for Testing

Prerequisites:
- 2 Users
- 1 Course with messages enabled

1. Log in to Artemis
2. Navigate to Course messages tab
3. Prepare a message including multiple external URLs (e.g.  `https://github.com/ls1intum/Artemis/pull/6616`)
4. Then send the message
5. You should see below the message a preview of the URLs **but without an preview image.**

#### Performance Review
- [x] I confirm that the client changes (in particular related to REST calls and UI responsiveness) are implemented with a very good performance 
#### Code Review
- [x] Code Review 1
- [x] Code Review 2
#### Manual Tests
- [x] Test 1
- [x] Test 2

### Screenshots

When multiple links in a message:
Screenshot 2023-07-19 at 02 00 29

### CODE REVIEW

**Review by tobias-lippert — APPROVED** [review]
Code LGTM

File: src/main/webapp/app/shared/link-preview/components/link-preview/link-preview.component.html [hunk 1]
```diff
@@ -4,11 +4,13 @@
         <!-- Link preview template -->
         <div class="preview-card">
             <div class="preview-content">
-                <h2 class="preview-title">{{ linkPreview.title }}</h2>
+                <a [href]="linkPreview.url" target="_blank" rel="noopener noreferrer"
+                    ><h2 class="preview-title">{{ linkPreview.title }}</h2></a
+                >
```
Comment (bassner) [inline src/main/webapp/app/shared/link-preview/components/link-preview/link-preview.component.html:9]: Formatting :)
```suggestion
                <a [href]="linkPreview.url" target="_blank" rel="noopener noreferrer">
                    <h2 class="preview-title">{{ linkPreview.title }}</h2>
                </a>
```

**Review by bassner — CHANGES_REQUESTED** [review]

**Review by tobias-lippert — APPROVED** [review]
Code Reapprove

**Review by DominikRemo — APPROVED** [review]
Tested in testing session on ts1. Works as expected 🚀

**Review by nityanandaz — APPROVED** [review]
I confirm that it only shows an image for one link preview (ts1)

**Pull request merged**